code/clue.js

```diff
@@ -3,33 +3,181 @@
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
+  color: 'Green',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'knife'
+}
+
+const mrMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'Yellow',
+  description: 'x',
+  age: 1,
+  image: 'assets/mustard.png',
+  occupation: 'x',
+  favouriteWeapon: 'rope'
+}
+
+const mrPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'Plum',
+  description: 'x',
+  age: 2,
+  image: 'assets/plum.png',
+  occupation: 'x',
+  favouriteWeapon: 'candlestick'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'Red',
+  description: 'x',
+  age: 1,
+  image: 'assets/scarlet.png',
+  occupation: 'x',
+  favouriteWeapon: 'bat'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'White',
+  description: 'x',
+  age: 1,
+  image: 'assets/white.png',
+  occupation: 'x',
+  favouriteWeapon: 'poison'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'Brown',
+  description: 'x',
+  age: 1,
+  image: 'assets/peacock.png',
+  occupation: 'x',
+  favouriteWeapon: 'axe'
 }
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
   name: 'rope',
-  weight: 10
+  weight: 10,
+  id: 'rope'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const knife = {
+  name: 'knife',
+  weight: 10,
+  id: 'knife'
+}
+
+const candlestick = {
+  name: 'candlestick',
+  weight: 10,
+  id: 'candlestick'
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 10,
+  id: 'dumbbell'
+}
+
+const poison = {
+  name: 'poison',
+  weight: 10,
+  id: 'poison'
+}
+
+const axe = {
+  name: 'axe',
+  weight: 10,
+  id: 'axe'
+}
+
+const bat = {
+  name: 'bat',
+  weight: 10,
+  id: 'bat'
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 10,
+  id: 'trophy'
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 10,
+  id: 'pistol'
+}
 
+// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const diningRoom = 'Dining Room'
+const conservatory = 'Conservatory'
+const kitchen = 'Kitchen'
+const study = 'Study'
+const library = 'Library'
+const billiardRoom = 'billiard Room'
+const lounge = 'Lounge'
+const ballRoom = 'Ballroom'
+const hall = 'Hall'
+const spa = 'Spa'
+const livingRoom = 'Living Room'
+const observatory = 'Observatory'
+const theater = 'Theater'
+const guestHouse = 'Guest House'
+const patio = 'Patio'
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  mrPlum,
+  mrsPeacock,
+  mrMustard,
+  missScarlet
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
```

Comment:
Not sure if it's preferred to write array items vertically or horizontally but I think it saves a lot of space if you write as a row if there are not so many items like in the rooms array.

Reply:
I would agree with Max over here - it's far more clean to format array in a away that single element has separate line